relay-general/src/store/normalize.rs

```diff
@@ -749,7 +750,9 @@ pub fn light_normalize_event(
             config.max_secs_in_future,
         )?; // Timestamps are core in the metrics extraction
         normalize_event_tags(event)?; // Tags are added to every metric
-        normalize_device_class(event);
+        if config.device_class_synthesis_config {
+            normalize_device_class(event);
+        }
```

Comment:
I believe this should be part of the store normalization instead of light normalization, but let's leave it as is for now. Could you add a `TODO` here to move it to store normalization?